src/controllers/check.company.controller.ts

```diff
@@ -1,17 +1,45 @@
 import { NextFunction, Request, Response } from "express";
+import logger from "../logger";
+import { PTFCompanyProfile } from "../model/company.profile";
+import { getCompanyProfile } from "../client/apiclient";
+import * as errorMessages from "../model/error.messages";
+import * as templatePaths from "../model/template.paths";
+import * as sessionService from "../services/session.service";
 
 export const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 
-    const scenario = { company: { PTFRequested: "Yes", accountsDue: "01 January 2020",
-            accountsStatus: "overdue", address: "25 No Street, Nowhere", companyType: "Private Limited Company",
-            csDue: "02 January 2020", csStatus: "overdue", incorporationDate: "01 January 2000",
-            name: "Test Company LTD", number: "00001111", status: "Active" }};
+    const companyNumber: string = sessionService.getCompanyInContext(req.chSession);
```

Comment:
This looks a bit odd - if a company is in context and stored in the session then why do we need to go and retrieve it again? I'm guessing that 'getCompanyInContext' on the session object is wrong and it should be named 'getCompanyNumberInContext'?

Reply:
Yes it's confusing - will change this to say number